lib/distance.c

```diff
@@ -199,8 +199,8 @@ distance_levenshtein(struct sr_thread *thread1,
     int n = frame_count2 + 1;
 
     // store only two last rows and columns instead of whole 2D array
-    SR_ASSERT(n <= SIZE_MAX - 1);
-    SR_ASSERT(m <= SIZE_MAX - (n + 1));
+    SR_ASSERT(n <= INT32_MAX - 1);
```

Comment:
To be honest, I think this assertion is coming a tad late, but I really don’t think we’ll ever get enough frames to cause an overflow.